thali/thalireplicationmanager.js

```diff
@@ -112,6 +113,28 @@ ThaliReplicationManager.prototype.stop = function () {
   });
 };
 
+/**
+* Returns the current device's id if available. If not, the provided callback
+* function is saved and called when the id does become available.
+* @param {Function} cb the callback which returns the current device's id.
+*/
+ThaliReplicationManager.prototype.getDeviceIdentity = function (cb) {
+  if(currentDeviceIdentity) {
+    cb(null, currentDeviceIdentity);
+  } else {
+    var cryptomanager = require('./thalicryptomanager');
```

Comment:
I wasn't referring to the function definition. I was referring to your use of require('./thalicryptomanager'). Generally it is considered good practice to put require statements at the top of the file. See http://justbuildsomething.com/node-js-best-practices/#2 for an example why.